SDL/CocoaBarItems.mm

```diff
@@ -366,31 +371,41 @@ -(NSMenu *)makeDebugMenu {
     [parent addItem:ignoreIllegalRWAction];
     [parent addItem:[NSMenuItem separatorItem]];
 
+    // Tags 1, 2, and 3 are taken above.
+
     NSMenuItem *loadSymbolMapAction = [[NSMenuItem alloc] initWithTitle:DESKTOPUI_LOCALIZED("Load Map File...") action:@selector(loadMapFile) keyEquivalent:@""];
     loadSymbolMapAction.target = self;
+    loadSymbolMapAction.tag = 4;
```

Comment:
These tags should really be an enum... much magic.

-[Unknown]

Reply:
Indeed, was just following Serena's pattern for now. Intend to make something cleaner here later.